src/module/actor/character/index.ts

```diff
@@ -102,6 +102,9 @@ class CharacterPF2e extends CreaturePF2e {
     class!: Embedded<ClassPF2e> | null;
     deity!: Embedded<DeityPF2e> | null;
 
+    // base ability scores, before any rule element breaks things
+    baseAbilityScores!: Abilities | null;
+
```

Comment:
This is unnecessary and can get acquired in the builder application

Reply:
There is no place that the after calculated ability scores are available. This is for builder mode too.

Reply:
How about the bases be set for each score in the abilities object?

Reply:
ok i'll do that